common/src/api/external/mod.rs

```diff
@@ -517,6 +517,8 @@ pub enum ResourceType {
     Silo,
     SiloUser,
     ConsoleSession,
+    GlobalImage,
+    GlobalImageList,
```

Comment:
removed in 6aa953e2, though I'm confused because [it looks like ConsoleSessionList is currently in there](https://github.com/oxidecomputer/omicron/blob/main/common/src/api/external/mod.rs#L520) ?

Reply:
You're right -- sorry!  I must have been looking at the wrong version when I posted that comment.  I did just check and I can build without `ConsoleSessionList`, so I will put up a separate PR to remove it.